data/json/recipes/recipe_deconstruction.json

```diff
@@ -506,6 +506,60 @@
     ]
   },
   {
+    "result": "makeshift_oscilloscope",
```

Comment:
Deconstruction recipes are for those recipes that are uncrafted in some way that doesn't resemble crafting in inverse too much.
For example, rag uncrafting doesn't require knitting needles, but takes a ton of time.

For items that have sane recipes that can be followed in the opposite way to recover what was used to craft them, it's better to just make the original recipe reversible.